packages/kbn-react-field/package.json

```diff
@@ -2,5 +2,6 @@
   "name": "@kbn/react-field",
   "version": "1.0.0",
   "license": "SSPL-1.0 OR Elastic License 2.0",
-  "private": true
+  "private": true,
+  "sideEffects": false
```

Comment:
**note:** required to tree-shake the package and prevent importing all the exported and unused components into our async chunks.